docs/_source/tutorials_and_integrations/tutorials/feedback/monitoring-bias-ethics-rlhf.ipynb

```diff
@@ -0,0 +1,628 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 👀 Monitoring and curating LLMs for ethics and bias using RLHF"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this tutorial, we explore a method to address `bias` in language models by enhancing the input data using `Reinforcement Learning with Human Feedback`. This approach aligns with ethical considerations by actively involving human judgment to guide the learning process, ensuring a more balanced and fair representation in the model's outputs.\n",
+    "\n",
+    "The steps are as follows:\n",
+    "\n",
+    "1. Test our LLM using [Giskard](https://www.giskard.ai/) and analyze the results\n",
+    "2. Create an Argilla `Feedback Dataset` according to the outputs of Giskard\n",
```

Comment:
Same here, I would write `FeedbackDataset`. As we are writing it as code, it should match our class name